spec/project/project_requires_spec.rb

```diff
@@ -1,8 +1,19 @@
 RSpec.describe 'Project requires' do
+  # rubocop:disable RSpec/MultipleExpectations
   it 'alphabetizes cop requires' do
-    source   = SpecHelper::ROOT.join('lib', 'rubocop-rspec.rb').read
-    requires = source.split("\n").grep(%r{rubocop/cop/rspec/[^(?:cop)]})
+    source   = SpecHelper::ROOT.join('lib', 'rubocop', 'cop', 'rspec_cops.rb')
+    captures = source.read.scan(%r{^(require_relative 'rspec/(.*?/)?(.*?)')$})
```

Comment:
Perhaps using named captures will make the `sort_by` block a bit clearer?

Reply:
Hm. `scan` and `grep` both ignore named captures. I think doing a manual map and match for the named captures doesn't feel necessary to me.